module/input/Microphone/README.md

```diff
@@ -0,0 +1,14 @@
+Microphone
+==========
+
+## Description
+
```

Comment:
Something like, "starts a subprocess and emits messages when the subprocess outputs something to `stdout`".